.changesets/span-api-data-filtering.md

```diff
@@ -0,0 +1,5 @@
+---
+bump: "patch"
```

Comment:
Shouldn't this be a minor bump?

Reply:
It's for the experimental Span API. 99.999% of users won't use this. So I wouldn't give a minor version bump for something that's mostly unused at the moment.